teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def intializate(specialization, age, name = 'Unknown', parent_permission: true)
+    uper(age, name, parent_permission)
```

Comment:
- [ ] You have a spelling error on this line, it should be `super` no `uper` which you currently have, please kindly make the necessary correction and resubmit for aproval :+1: :sparkles: